style.css

```diff
@@ -136,12 +263,34 @@ body {
   position: relative;
   top: 70px;
   left: -10px;
-  margin: 0 90px 41px 8px;
-  margin-top: 30px;
-  margin-bottom: 41px;
+  margin: 30px 90px 41px 8px;
   font: 500 22px/26px Roboto, arial, helvetica, sans-serif;
 }
 
+@media (max-width: 1366px) {
+  .main__subtitle {
+    margin: -14px 0 17px -70px;
+    left: 238px;
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 1024px) {
+  .main__subtitle {
+    margin: -15px 186px 17px -230px;
+    left: 238px;
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 768px) {
+  .main__subtitle {
+    margin: -54px 13px 27px 22px;
+    left: 2px;
+    font: 500 16px/19px Roboto,arial,helvetica,sans-serif;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6362272)